Sming/Core/Network/Http/HttpServerConnection.h

```diff
@@ -120,6 +125,8 @@ class HttpServerConnection : public HttpConnection
 
 	const BodyParsers* bodyParsers = nullptr;	///< const reference ensures we cannot modify map, only look stuff up
 	HttpBodyParserDelegate bodyParser = nullptr; ///< Active body parser for this message, if any
+	bool closeOnBadBodyContent = false;
```

Comment:
Please, rename `closeOnBadBodyContent` to `closeOnContentError`.

Reply:
done